lib/response.js

```diff
@@ -56,8 +56,14 @@ var charsetRegExp = /;\s*charset\s*=/;
  * @public
  */
 
-res.status = function status(code) {
-  this.statusCode = code;
+res.status = function status(statusCode) {
+  // check that status code is valid
+  statusCode = parseInt(statusCode, 10);
```

Comment:
Node/express currently works with any numeric string as a status code (node does `statusCode |= 0`) under the hood, seems like only accepting numbers could be an annoying change for anyone relying on it. Thoughts?